### ISSUE TITLE
dash.wav and dot.wav

### ISSUE THREAD

**Issue by tlanc007** [kirkshoop/twitter#4]
The CMake file and main.cpp make references to dash.wav and dot.wav.  But these files aren't included in the repo.

**Comment by kirkshoop** [comment]
Yes. I used files from my mac that I do not have the rights to. Just pick any short notification wav and name it dash.wav and dot.wav

**Comment by leira** [comment]
I'm on Mac too, where can I find these 2 files?

**Comment by tlanc007** [comment]
As Kirk replied to my query you will need to substitute your own files.

One way is to copy over two short audio files and call one dash.wav and the other dot.wav.

For me,  the audio wasn't important, so I just commented out the references entirely.

**Comment by leira** [comment]
Yeah, that was what I did too. I misunderstood Kirk, I thought he meant these 2 files were from macOS, so I asked.
